en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -1,361 +1,104 @@
         </td>
     </tr>
     <tr>
         <td>
-            Publisher
+            Traffic Manager Profile
         </td>
         <td>
-            Enables API providers to easily publish their APIs, share documentation, provision API keys, and gather feedback on API features, quality, and usage.
-        </td>
-    </tr>
-    <tr>
-        <td>
-            Developer Portal
-        </td>
-        <td>
-            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
+            Only starts the Traffic Manager component.
         </td>
     </tr>
 </table>
 
-## Databases for API-M components
+## API-M Components
 
-WSO2 API Manager components use the databases given below. Find out more about [databases in API-M]({{base_path}}/install-and-setup/setup/setting-up-databases/overview).
+Listed below are the five components of the API-M runtime and its function. 
 
 <table>
-<thead>
-<tr class="header">
-<th><br />
-</th>
-<th><p><strong>API Manager<br />
-database</strong></p>
-<p><code>              apimgtdb             </code></p>
-<p><code>              WSO2_AM_DB             </code></p></th>
-<th><p><strong>Shared Database</strong></p>
-<p><code>                                            shareddb                           </code></p>
-<p><code>              WSO2_SHARED_DB             </code></p></th>
-
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p><strong>Publisher</strong></p></td>
-<td><p>Used</p></td>
-<td><p>Used</p></td>
-
-</tr>
-<tr class="even">
-<td><p><strong>Developer Portal</strong></p></td>
-<td><p>Used</p></td>
-<td><p>Used</p></td>
-
-</tr>
-<tr class="odd">
-<td><p><strong>Key Manager</strong></p></td>
-<td><p>Used</p></td>
-<td><p>Used</p></td>
-</tr>
-<tr class="even">
-<td><p><strong>Gateway</strong></p></td>
-<td><p>Not used</p></td>
-<td><p>Used (in multi-tenancy mode/ in multiple gateway mode when Google Analytics is used)</p></td>
-
-</tr>
-<tr class="odd">
-<td><strong>Traffic Manager</strong></td>
-<td>Used</td>
-<td>Used</td>
-
-</tr>
-</tbody>
-</table>
-
-<!--
-## Understanding the distributed deployment
-
-!!! Note
-    Work in progress!
-
-When we consider a distributed deployment of WSO2 API Manager, we have the option of separating the five components and clustering each component as needed. Let's look more closely at how the API Manager components are deployed separately.
-
-In the following diagram, the five components are set up in a distributed deployment, and the five databases are connected to the relevant components respectively. The entire setup is also fronted by a load balancer.
-
-[![Understanding the distributed deployment]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)
-
-??? "Click here for more details on default communication ports among components."
```

Comment:
Why do we drop this sectio "Click here for more details on default communication ports among components." ?

Reply:
@isharac - Can we get this content for API-M 4 as well.. The previous content was outdated for API-M, hence I removed them.

Reply:
Will send a separate PR for this one when the diagrams are ready.